docs/static/katex/README.md

```diff
@@ -84,7 +84,7 @@ Learn more about using KaTeX [on the website](https://katex.org)!
 
 ## Contributing
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
+See [CONTRIBUTING.md](../../../CONTRIBUTING.md)
```

Comment:
@0xbrw - I don't think this link is supposed to point to our Contributing page; this looks to be a clone of this page:
https://github.com/KaTeX/KaTeX/blob/main/README.md

I'm not sure why it lives in our repo, but in any case, we should aim it at the KaTeX readme